client/src/rubin/nublado/client/__init__.py

```diff
@@ -0,0 +1,3 @@
+from .nubladoclient import NubladoClient
+
+__all__ = ["NubladoClient"]
```

Comment:
There are a lot of modules in the Nublado client. I wonder if it's be more user friendly to make some of these modules private and import their APIs into a higher level like or, or at most have a `rubin.nublado.client`, `rubin.nublado.client.exceptions` and `rubin.nublado.client.models` spaces in the public API?

Reply:
I think in general a simplified hierarchy exposed to the user is a good idea, although I'll need to think about how to structure that.  We also want `.testing.MockJupyter` exposed, probably there, because service authors are going to want that piece.  But all the ancillary stuff in `.testing.jupyter` probably should stay hidden.  Maybe lifting the mock into `.testing`, the exception from the extension model into `.exceptions`. and leaving the models pretty much alone is the right thing?

Anyway, yes, just need to think about exactly how.

Reply:
Done.  Not sure if it's much better, but import names are slightly shorter, anyway.